docs/api_miniscreen.rst

```diff
@@ -0,0 +1,14 @@
+======================================
+API - pi-top [4] Miniscreen
+======================================
+
+pi-top [4]'s case includes a miniscreen module, composed by a OLED screen and 4 buttons.
+On pi-topOS, the OLED and buttons display an interactive menu, with useful information about
```

Comment:
I think we should:
* Mention the package here, so that people can look into it themselves
* Explain how the OLED and button APIs work in parallel with the system menu
* Explain how users can return to the main OLED screen from their code

Reply:
This appears to be captured in the OLED section, but the same goes for the buttons, so we should probably include a high level note about this here.

Reply:
Also add to FAQs